tests/shutdown/svirt_upload_assets.pm

```diff
@@ -38,6 +39,8 @@ sub extract_assets {
 }
 
 sub run {
+    # Skip if the os-autoinst backend implements this feature
+    return 1 if backend::svirt->can('do_extract_assets');
```

Comment:
As pointed out by @Martchus [here](https://github.com/os-autoinst/os-autoinst/pull/1895#discussion_r783264756), this will return true because it's implemented n the baseclass and `can()` will check parent classes.
`if defined &backend::svirt::do_extract_assets` should work instead.